### PR TITLE
add User class with fields

### PR DESCRIPTION
**Pull request by EmilijaSvirsk**

### CODE REVIEW

File: EncounterMe/EncounterMe/User.cs [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+using System.Collections.Generic;
+using System.Text;
+using System.Security.Cryptography;
+using System.Linq;
+
+namespace EncounterMe
+{
+    class User
+    {
+        private string name;
```
Comment (AdamBeresnev) [inline EncounterMe/EncounterMe/User.cs:11]: You should probably move the variable declarations to the top of the class for clarity

**Review by krtrinkunas — COMMENTED** [review]
looks good

**Comment by AdamBeresnev** [comment]
I see no issues

**Comment by EmilijaSvirsk** [comment]
Looks great

**Comment by AdamBeresnev** [comment]
I see no issues

**Comment by DominykasVi** [comment]
Looks good, nice touch with hashing

**Pull request merged**